api/src/main/java/org/apache/cloudstack/config/ApiServiceConfiguration.java

```diff
@@ -21,7 +21,7 @@
 
 public class ApiServiceConfiguration implements Configurable {
     public static final ConfigKey<String> ManagementServerAddresses = new ConfigKey<String>("Advanced", String.class, "host", "localhost", "The ip address of management server. This can also accept comma separated addresses.", true);
-    public static final ConfigKey<String> ApiServletPath = new ConfigKey<String>("Advanced", String.class, "endpointe.url", "http://localhost:8080/client/api",
+    public static final ConfigKey<String> ApiServletPath = new ConfigKey<String>("Advanced", String.class, "endpoint.url", "http://localhost:8080/client/api",
```

Comment:
this change may not be required, as it is handled in schema upgrade.

Reply:
I think, if we go ahead with this, it must be in both the schema change and in this object, @sureshanaparti. Otherwise both will end up in the table.

Reply:
> I think, if we go ahead with this, it must be in both the schema change and in this object, @sureshanaparti. Otherwise both will end up in the table.

ok @DaanHoogland